src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,54 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../react-app-env';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs, selectedTabId, onTabSelected,
+}) => {
+  const currentTab = tabs.find(tab => tab.id === selectedTabId);
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map((tab, index) => (
+              <li
+                className={classNames({
+                  'is-active': selectedTabId === tab.id
+                   || (!currentTab && !index),
```

Comment:
```suggestion
                  'is-active': selectedTabId === tab.id,
```
Avoid this by adding default value to your currentTab (tab[0])